examples/keycloak-config-cli-using-spire/java-spiffe-helper.yaml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
One more change that may make the pr easier to use...

There is a a feature of the chart here that could be useful:
https://artifacthub.io/packages/helm/bitnami/keycloak#deploy-extra-resources

It would let you move this configmap into the keycloak-values.yaml file and remove it as a needed step in the README?

Reply:
Yep. Tested and works as well. Wasn't aware of that key. Thanks again!